src/scripts/tweaks/unsticky_tab_bar.js

```diff
@@ -0,0 +1,10 @@
+import { keyToCss } from '../../util/css_map.js';
+import { buildStyle } from '../../util/interface.js';
+
+const styleElement = buildStyle(`
+  ${keyToCss('timelineHeader')}${keyToCss('sticky')} { position: static; }
```

Comment:
would be nice to remove the slide effect since that's visible if you scroll to the top of the page too quickly

```suggestion
  ${keyToCss('timelineHeader')}${keyToCss('sticky')} { position: static; transform: none; }
```